electricitylci/egrid_facilities.py

```diff
@@ -2,13 +2,10 @@
 import stewi
 from os.path import join
 from electricitylci.globals import data_dir
-from electricitylci.model_config import (
-    egrid_year,
-    min_plant_percent_generation_from_primary_fuel_category
-)
+from electricitylci.model_config import model_specs
 
 # get egrid facility file from stewi
-egrid_facilities = stewi.getInventoryFacilities("eGRID", egrid_year)
+egrid_facilities = stewi.getInventoryFacilities("eGRID", model_specs.egrid_year)
```

Comment:
I don't know if it's a dependency version thing or something, but I'm getting errors here - '`dict' object has no attribute 'egrid_year'`. When I change it to model_specs["egrid_year"], it's fine.

Reply:
Are  you running main.py? or from the command line?

Reply:
Neither? I've got a separate script to generate some data using some direct function calls. I get the error from calling create_generation_process_df, which imports generation.py, which imports egrid_facilities.py

```
import pandas as pd
import electricitylci
import lciafmt
import electricitylci.aggregation_selector as agg_select
import electricitylci.model_config as config
config.model_specs=config.load_model_specs('ELCI_1')
#import electricitylci.analysis.apply_mixes as mixes
from electricitylci.generation import create_generation_process_df
```

Reply:
just noticed that I called load_model_specs as opposed to build model class.

Reply:
need to use build_model_class instead of load_model_specs, and then I think you should be fine.

Reply:
Based on that, it might be worth making load_model_specs  an internal function for build_model_class. Or you could also rename load_model_specs to _load_model_specs so that it's ignored by most IDEs. Just not sure if their is a use case for model_config.load_model_specs().

Reply:
@bl-young @jump2conclusionsmatt  Its appears this was added in @bl-young's recent commit. Can we merge this now?